### ISSUE TITLE
Problem in a project with two targets, generating a framework for iOS and Apple TV

### ISSUE THREAD

**Issue by talesp** [CocoaPods/CocoaPods#4928]
I'm trying to create a project with two targets, one generating a framework for iOS and the other generating a framework for Apple TV.

Here is the Podfile:

```ruby
workspace "CSGitHub.xcworkspace"
project "Shared/Shared.xcodeproj"

def shared_pods
    pod "Freddy", "~> 2.0.2"
    pod "Moya/RxSwift"
    pod "Nuke"
end

abstract_target "Shared" do
    
    use_frameworks!
    inhibit_all_warnings!
    
    target "SharediOS" do
        platform :ios, '9.0'
        
        shared_pods
    end

    target "SharedTV" do
        platform :tvos, '9.2'
        
        shared_pods
    end

end
```

If I run `pod install` like this, it will generate targets like this

![captura de tela 2016-02-23 00 55 34](https://cloud.githubusercontent.com/assets/221485/13241479/29ad21e0-d9ca-11e5-87f9-6f1a1e80a68c.png)

If I try to compile like this, I get this error:

```
ld: warning: directory not found for option '-F/Users/talesp/Library/Developer/Xcode/DerivedData/CSGitHub-chhyzynmakhwybfurbidfrwoahko/Build/Products/Debug-iphonesimulator/Pods-SharediOS'
ld: framework not found Alamofire
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

I tried to comment one of the targets inside the `abstract_target` block, and everything went OK, and the targets inside the `Pods` project looked like this:

![captura de tela 2016-02-23 00 57 23](https://cloud.githubusercontent.com/assets/221485/13241593/34a55436-d9cb-11e5-9c29-b15abe27cf6f.png)

Searching though the project settings, I found that, besides the fact that when both targets are active it adds the prefix `Pod-MyTargetName-` (for example, `Pods-SharediOS-Alamofire`) to the name of the shared framework, the inherit setting in my project/target still references the original framework name (in this case, `Alamofire`.

So, I think when there are multiple targets generating frameworks for two different platforms, the parameter `OTHER_LDFLAGS` in the `*.xcconfig` files inside `Pods/Target Support Files/Pod-MyTarget` should reflect the correct framework name.

Here is a link for a sample project https://www.dropbox.com/s/d8en5ln5zao39o1/CSGitHub.zip?dl=0

**Comment by segiddins** [comment]
This might be fixed in @mrackwitz 's open PR

**Comment by talesp** [comment]
Cool, do you have any plan to release a new beta including this PR?

**Comment by fguchelaar** [comment]
This seems like a duplicate to #4920 and #4926, which are both resolved in 1.0.0.beta.4.

**Comment by mrackwitz** [comment]
Yes, the latest prerelease contains a fix for this issue.
